.travis.yml

```diff
@@ -67,21 +68,22 @@ after_failure:
   - more /tmp/sanitizer* | cat          # dump all sanitizer results
 
 before_deploy:
-  - find . -name '*.pyc' -delete
-  - for arch in x86-64 sandybridge haswell; do
+  - for arch in core2 sandybridge haswell; do
       export CXXARCHFLAGS="-march=$arch";
       ./container_build.py clean;
       ./container_build.py bess;
       ./container_build.py kmod_buildtest;
-      tar zcf bess-$arch-linux.tar.gz bessctl/* bin/* pybess/* core/bessd core/kmod/bess.ko core/modules/*.so;
+      find . -name '*.pyc' -delete;
+      (cd .. && tar zcf bess/bess-$arch-linux.tar.gz bess/bessctl/* bess/bin/* bess/pybess/* bess/core/kmod/* bess/core/snbuf_layout.h bess/core/bessd bess/core/modules/*.so);
+      fpm -s tar -t deb -v ${TRAVIS_TAG#v} -n bess -d python -d python-pip -d python-scapy -d libgraph-easy-perl -m "Chang Lan <clan@eecs.berkeley.edu>" --url "https://github.com/NetSys/bess" --prefix "/usr/local/" --after-install=env/after_install.sh --before-remove=env/before_remove.sh -p bess-$arch-linux.deb bess-$arch-linux.tar.gz
```

Comment:
Could you change the contact to the bess maintainers address (bess (at) nefeli (dot) io)?